client/dashboard/index.js

```diff
@@ -1,17 +1,24 @@
 /**
  * External dependencies
  */
-import { Component } from '@wordpress/element';
+import { Component, Suspense, lazy } from '@wordpress/element';
 import { compose } from '@wordpress/compose';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
-import CustomizableDashboard from './customizable';
-import ProfileWizard from '../profile-wizard';
 import withSelect from 'wc-api/with-select';
 import { isOnboardingEnabled } from 'dashboard/utils';
+import { Spinner } from '@woocommerce/components';
+
+const CustomizableDashboard = lazy( () =>
```

Comment:
This prevents the styles for hiding the dashboard when the profiler is loaded added by `.woocommerce-admin-full-screen`.  Maybe util styles like that could be preloaded beforehand.

<img width="889" alt="Screen Shot 2020-04-28 at 1 07 16 PM" src="https://user-images.githubusercontent.com/10561050/80481847-e548bc80-895b-11ea-9a44-ce29ab70a75d.png">

Reply:
Also, should we center the loading spinner in the above screen?

Reply:
> This prevents the styles for hiding the dashboard when the profiler is loaded added by `.woocommerce-admin-full-screen`. Maybe util styles like that could be preloaded beforehand.

I don't think the styles aren't working - the class is added on component mount which is now delayed because of the lazy load. I had to throttle my browser to 3G to experience a lengthy delay.

I'll experiment with adding the class on the PHP side, but I'm not sure it will solve the problem completely.

> Also, should we center the loading spinner in the above screen?

I think ideally we'd use something other than spinners for these lazy loaded components. I'll see if it's straightforward to use the placeholder treatments from each instead - though that might eat into our bundle size savings a little (if possible).

Reply:
>I don't think the styles aren't working - the class is added on component mount which is now delayed because of the lazy load. I had to throttle my browser to 3G to experience a lengthy delay.

This happens to me locally when loading the profiler every time.  It's only for a second but the dashboard is visible for that brief moment.  I forgot that we switched these styles over to `woocommerce-admin-is-loading` which is added to the body class server-side.  If we can make sure that style is loaded beforehand, that should resolve it.

But we can tackle this in a follow-up so we don't block this PR further.

>I'll see if it's straightforward to use the placeholder treatments from each instead - though that might eat into our bundle size savings a little (if possible).

This would be awesome if we can find a reasonably lightweight solution.